llarp/router/router.cpp

```diff
@@ -421,36 +424,25 @@ namespace llarp
       m_isServiceNode = true;
     }
 
+    /// build a set of  strictConnectPubkeys (
+    /// TODO: make this consistent with config -- do we support multiple strict connections
```

Comment:
yes we do support multiple strict connections

Reply:
We weren't in this codepath. I suspect it wasn't actually possible, although code downstream supported it (e.g. taking a `std::set<>` of strict connect keys).